Sources/SwifterSwift/UIKit/UICollectionViewExtensions.swift

```diff
@@ -147,6 +147,22 @@ public extension UICollectionView {
         register(UINib(nibName: identifier, bundle: bundle), forCellWithReuseIdentifier: identifier)
     }
 
+    /// SwifterSwift: Safely scroll to possibly invalid IndexPath
+    ///
+    /// - Parameters:
+    ///   - indexPath: Target IndexPath to scroll to
+    ///   - scrollPosition: Scroll position
+    ///   - animated: Whether to animate or not
+    func safeScrollToItem(at indexPath: IndexPath, at scrollPosition: UICollectionView.ScrollPosition, animated: Bool) {
+        guard indexPath.row >= 0 &&
```

Comment:
Even though they're technically the same, `.row` is designed for `UITableView`s and `.item` is for `UICollectionView`.
```suggestion
        guard indexPath.item >= 0 &&
```